.github/scripts/test_trymerge.py

```diff
@@ -205,7 +205,6 @@ def mocked_read_merge_rules(repo: Any, org: str, project: str) -> List[MergeRule
             approved_by=["pytorch/metamates", "ngimel"],
             mandatory_checks_name=[
                 "Lint",
-                "Facebook CLA Check",
```

Comment:
As this is a mock value, this is probably ok.  The test you see fails because it caches an old PR https://github.com/pytorch/pytorch/pull/79694 where `Facebook CLA Check` was still in used?  Another way is to choose a newer example I think

Reply:
Yeah, but I'm unsure how to put the new pr in. Also a good example is kindof hard to find without making it yourself.